frontend/elements/src/components/error/styles.sass

```diff
@@ -8,14 +8,16 @@
   color: variables.$error-color
   background: variables.$background-color
 
-  padding: .25rem
+  padding: .50rem 1.5rem .50rem .50rem 
   margin: variables.$item-margin
   min-height: variables.$item-height
 
   display: flex
-  align-items: center
+  align-items: start
   box-sizing: border-box
 
+  line-height: 20px
```

Comment:
Wouldn't it make more sense to set a unitless value or `normal` to avoid unexpected results?

Reply:
And what about a unitless value that works for now? Also there actually is a shadow-part for the error text, it's called `error-text`.